### PR TITLE
Function chaining

### PR DESCRIPTION
**Pull request by otuyamicah**
add function chaining

### CODE REVIEW

File: execution_stack.js [hunk 1]
```diff
@@ -1,24 +1,38 @@
+var name = "micah"
+
+
+
 function one(){
-    console.log('i am function one')
+    var x = "hi"
```
Comment (Pedestal19) [inline execution_stack.js]: Please can you rename this variable?

Comment (Pedestal19) [inline execution_stack.js]: Please can you rename this variable?

File: execution_stack.js [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
+    two()
 }
-one()
+
 
 
 
 function two(){
-    util()
+    var y = "howfar"
```
Comment (Pedestal19) [inline execution_stack.js]: rename this too

File: execution_stack.js [hunk 3]
```diff
@@ -1,24 +1,38 @@
...
+    console.log(y + " micah")
+    three()
 }
-two()
 
 
 
 function three(){
-    console.log('i am function three')  
+    var z = "how e be"
```
Comment (Pedestal19) [inline execution_stack.js]: rename this too

**Review by Pedestal19 — CHANGES_REQUESTED** [review]
Please refactor

**Review by Pedestal19 — CHANGES_REQUESTED** [review]
Please refactor

**Review by Pedestal19 — CHANGES_REQUESTED** [review]
I meant refactoring the variables

**Review by Pedestal19 — CHANGES_REQUESTED** [review]
rtefactor variable to meaningful names

**Review by Pedestal19 — APPROVED** [review]

**Comment by otuyamicah** [comment]
refactored variables

**Pull request merged**